subsys/bluetooth/controller/ll_sw/lll_adv.h

```diff
@@ -14,7 +14,7 @@ struct lll_adv_iso_stream {
 	memq_link_t *link_tx_free;
 
 	/* Downstream last packet sequence number */
-	uint16_t pkt_seq_num;
+	uint64_t pkt_seq_num:39;
```

Comment:
Packet Sequence Number in the HCI ISO Data is 16-bit. 

ISO implementation in the Controller is experimental and is missing ISOAL integration which shall implement the Packet Sequence Number supplied downstream from the Host to Controller.

In the current implementation, the use of `pkt_seq_num` in hci.c is temporary. Suggest changing the local `pkt_seq_num` in hci.c to `uint16_t` instead and fix the implementation that check for stale `stream->pkt_seq_num`.

Reply:
Understood. The ISOAL integration and pending API change [#43859](https://github.com/zephyrproject-rtos/zephyr/issues/43859) imply this PR is of little value.  It will be withdrawn.